src/Modal.css

```diff
@@ -0,0 +1,49 @@
+.Modal__overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.65);
+}
+
+.Modal__content {
```

Comment:
the modal body needs to become scrollable if you have a sufficiently small device you can't click the dismiss button.
![image](https://user-images.githubusercontent.com/6399796/57178353-92324b80-6e66-11e9-9783-95550e68d52e.png)

Reply:
you can do this by setting the `max-height` to a certain percentage

Reply:
👍